### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by abdulhamiid**
Hi, 👋

I have implemented the project requirements given in the below link:

https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md
DELETE
DELETE2
savepoint1
savepoint2
query
query2
query3

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,12 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
...
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- new queries
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT COUNT(neutered), neutered FROM animals GROUP BY neutered;
+SELECT MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight, species FROM animals GROUP BY species;
+SELECT AVG(escape_attempts) AS avg_escape_attempts, species FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species
```
Comment (VICTOR-CHUKWU) [inline queries.sql:39]: - [ ] Kindly add your transaction queries into your queries.sql, I can see you did it from the screenshots you put in your PR
```
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; 

   SELECT species from animals; 
   ROLLBACK;

   SELECT species from animals;
```

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @abdulhamiid  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (abdulhamiid) [reply queries.sql:39]: Alright, thanks for the feedback

File: data.sql [hunk 2]
```diff
@@ -3,4 +3,14 @@
...
+(4, 'Devimon', '2017-05-12', 5, TRUE, 11);
+
+-- TABLE update
+
+INSERT INTO animals VALUES (5, 'Charmander', '2020-02-08', 0, FALSE, -11),
+(6, 'Plantmon', '2021-11-15', 2, TRUE, -5.7),
+(7, 'Squirtle', '1993-6-02', 3, FALSE, -12.13),
+(8, 'Angemon', '2005-06-12', 1, TRUE, -45),
+(9, 'Boarmon', '2005-06-07', 7, TRUE, 20.4);
+(10, 'Blossom', '1998-10-13', 3, TRUE, -17),
```
Comment (BenMukebo) [inline data.sql]: - [OPTIONAL] Kindly make sure you really understand this 👇🏻 
  - Insert the following data: Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times. Let's change the `weight_kg` -17 value, Your code should look like:

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Blossom', '1998/10/13', '3', 'true', '17.0');
```

**Review by BenMukebo — APPROVED** [review]
### Hi @abdulhamiid, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (abdulhamiid) [reply data.sql]: Okay, thanks

**Pull request merged**